webapp/src/index.tsx

```diff
@@ -800,17 +774,29 @@ export default class Plugin {
             });
         });
 
-        this.registerWebSocketEvents(registry, store);
-        this.registerReconnectHandler(registry, store, () => {
-            logDebug('websocket reconnect handler');
-            if (!window.callsClient) {
-                logDebug('resetting state');
-                store.dispatch({
-                    type: UNINIT,
+        // A dummy React component so we can access webapp's
+        // WebSocket client through the provided hook. Just lovely.
```

Comment:
😂

Reply:
Will this cause us to depend on a new min server version?

Reply:
This has been there for a long time (before monorepo) so I think we are good. But of course I'd like to fix it properly one day (pass the client object directly to the init function) in which case we'll have to add some backwards compatibility check.